airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -52,15 +52,27 @@ definitions:
       - emitted_at
     properties:
       stream:
-        description: "the name of the stream for this record"
+        description: Name of the stream for this record
         type: string
       data:
-        description: "the record data"
+        description: The data
         type: object
         existingJavaType: com.fasterxml.jackson.databind.JsonNode
       emitted_at:
-        description: "when the data was emitted from the source. epoch in millisecond."
+        description: When the data was emitted from the source. (epoch in milliseconds)
         type: integer
+      metadata:
+        "$ref": "#/definitions/AirbyteRecordMessageMetadata"
+  AirbyteRecordMessageMetadata:
+    description: "Map of metadata fields. Keys and values must both be strings. Below are some well-known fields.\n
+     replication_batch_id: Identifies which sync job replicated the record\n
+     cdc_deleted_at: Timestamp when the record was deleted. null or unset if it has not been deleted. Only present for streams syncing data using CDC. (epoch in milliseconds)\n
+     cdc_updated_at: Timestamp when the record was updated. Only present for streams syncing data using CDC. (epoch in milliseconds)\n
```

Comment:
is there value in having the updated_at vs. just emitting the record?

Reply:
With CDC you know when the database got updated, not when the record was emitted.

Reply:
got it, so this is equivalent to `updated_at` even if it's not present on the db schema

Reply:
yup!